### PR TITLE
https://issues.redhat.com/browse/ACM-9063 12/13/23 MCE 2.4.2 Errata

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-9063

### CODE REVIEW

**Review by jc-berger — APPROVED** [review]
/lgtm

File: clusters/release_notes/errata.adoc [hunk 1]
```diff
@@ -5,6 +5,12 @@ For {mce-short}, the Errata updates are automatically applied when released.
 
 *Important:* For reference, link:https://access.redhat.com/errata/#/[Errata] links and GitHub numbers might be added to the content and used internally. Links that require access might not be available for the user. 
 
+== Errata 2.4.2
+
+* Delivers updates to one or more of the product container images.
+
+* Fixes an issue where the `AppliedManifestWork` resource was being deleted.
```
Comment (dockerymick) [inline clusters/release_notes/errata.adoc]:
```suggestion
* Fixes an issue where the `AppliedManifestWork` resource was being deleted. (link:https://issues.redhat.com/browse/ACM-8926[ACM-8926])
```

**Comment by cadawson** [comment]
Looks great, @dockerymick ! Thanks!

**Comment by dockerymick** [comment]
Thanks to you both for the approval! @jc-berger @cadawson 

Merging this PR

**Pull request merged**